_includes/sidebar.html

```diff
@@ -21,7 +21,10 @@
       {% for node in pages_list %}
         {% if node.title != null %}
           {% if node.layout == "page" %}
-            <a class="sidebar-nav-item{% if page.url == node.url %} active{% endif %}" href="{{ node.url }}">{{ node.title }}</a>
+            {% assign prefix = node.url | truncate: 10, '' %}
+            {% if prefix != "/blog/page" %}
```

Comment:
Don't use bare urls. Always prefix with `{{site.baseurl}}`. 
This theme can easily break if you don't (i.e. as soon as the baseurl is not '/').